high-score/lib/high_score.ex

```diff
@@ -0,0 +1,26 @@
+defmodule HighScore do
+  @default_initial_score 0
+  def new() do
+    %{}
+  end
+
+  def add_player(scores, name, score \\ @default_initial_score ) do
```

Comment:
```suggestion
  def add_player(scores, name, score \\ @default_initial_score) do
```
Remove the empty space

Reply:
Done